shared/slotutil/slotticker.go

```diff
@@ -38,8 +38,8 @@ func (s *SlotTicker) Done() {
 	}()
 }
 
-// GetSlotTicker is the constructor for SlotTicker.
-func GetSlotTicker(genesisTime time.Time, secondsPerSlot uint64) *SlotTicker {
+// New starts and returns a new SlotTicker instance.
+func New(genesisTime time.Time, secondsPerSlot uint64) *SlotTicker {
```

Comment:
I disagree with this change - `slotutil.New` does not convey information about what is being created. I would prefer `slotutil.NewSlotTicker`.